### PR TITLE
website link in github

### LINKED ISSUE #7: Setup firebase hosting
Just create a simple `index.html` file with just a h1 in it and then setup firebase hosting.

### PR DESCRIPTION
**Pull request by Gowrisankar1**
Firebase website link for GitHub is completed.

### CODE REVIEW

File: public/index.html
Comment (ajayesivan) [inline public/index.html:1]: @Gowrisankar1 Why does the deployment setup PR contain this much HTML change??

Comment (Gowrisankar1) [reply public/index.html:1]: I'm facing some issue in the firebase link apperance in GitHub. So I tried multiple ways to fix it that's why.

Comment (ajayesivan) [reply public/index.html:1]: The content of the index.html file won't affect the pipelines. Please remove the code added to the HTML file and just keep a single h1 tag as before. If you need any help with the process ping me on Discord.

**Review by ajayesivan — APPROVED** [review]
LGTM!

**Pull request merged**